openapi/code/entity.go

```diff
@@ -256,6 +256,15 @@ func (e *Entity) IsAllRequiredFieldsPrimitive() bool {
 	return true
 }
 
+func (e *Entity) IsAllRequiredFieldsJsonUnserialisable() bool {
```

Comment:
Shouldn't this be IsAllRequiredFieldsJsonSerializable? It returns true if for all required fields, IsJson is true.

Suggest adding a doccomment.